src/common/options/mds.yaml.in

```diff
@@ -73,6 +73,8 @@ options:
   default: 64_K
   services:
   - mds
+  flags:
+  - startup
```

Comment:
Put this into one separate commit to explain the reason ?

Reply:
Ok, makes sense.